packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -46,10 +46,10 @@ const main = () => {
             `  The PWA Kit CLI is used in NPM scripts so you can conveniently`,
```

Comment:
Do we still want this to be called "PWA Kit CLI"? (Also on line 32.)

Reply:
Good catch